linked_list/node.py

```diff
@@ -4,13 +4,26 @@ class Node(object):
     """
 
     def __init__(self, elem, next=None):
-        pass
+        self.elem = elem
+        self.next = next
 
     def __str__(self):
-        pass
+        if self.next:
+            return "Node({0}) > Node({1})".format(self.elem, self.next.elem)
+        else:
+            return "Node({}) > /".format(self.elem)
 
     def __eq__(self, other):
-        pass
+        if other == None:
+            return False
+        else:
+            return self.elem == other.elem
+        
+    def __ne__(self, other):
+        return not self.__eq__(other)
 
     def __repr__(self):
```

Comment:
Your `__str__` and `__repr__` currently do the same thing, instead of repeating yourself you can just have one call the other.